lib/matplotlib/text.py

```diff
@@ -1134,6 +1147,17 @@ def set_rotation(self, s):
         self._rotation = s
         self.stale = True
 
+    def set_rotation_transformed(self, t):
+        """
+        Set if the text rotation get transformed or not.
```

Comment:
Maybe better:
~~~
Whether rotations of the transform affect the text direction.
~~~